pkg/cli/tools/binary_tools.go

```diff
@@ -154,20 +151,6 @@ func DownloadToFolder(filepath string) error {
 		return err
 	}
 
-	// Create credentials to authenticate to repository
-	ds, err := credentials.NewStoreFromDocker(credentials.StoreOptions{
-		AllowPlaintextPut: true,
-	})
-	if err != nil {
-		return err
-	}
-
-	repo.Client = &auth.Client{
```

Comment:
If a client isn't specified, the oras client uses the default auth client (which uses the same default client/cache with an EmptyCredential instead of a set value) https://github.com/oras-project/oras-go/blob/062ed0e058f87a9661dfb5db41e8e72f74dfffe2/registry/remote/auth/client.go#L35